client/my-sites/site-settings/custom-content-types/index.jsx

```diff
@@ -99,7 +99,7 @@ class CustomContentTypes extends Component {
 		const { customContentTypesModuleActive } = this.props;
 
 		return (
-			<div className="custom-content-types__module-settings is-indented">
+			<div className="site-settings__child-settings">
```

Comment:
All of them - it should be fine if you bring back the `*-settings` class for each group.